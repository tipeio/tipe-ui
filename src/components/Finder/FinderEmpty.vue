<template>
  <div data-tipe-ui="TipeFinder">Empty</div>
</template>

<style lang="postcss" scoped>
[data-tipe-ui='TipeFinder'] {
  height: 100%;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 0.8125rem;
  color: var(--text-gray-normal);
  background-color: var(--gray-lightest);
}
</style>
