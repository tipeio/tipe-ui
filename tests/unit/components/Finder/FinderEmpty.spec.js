import { shallowMount } from '@vue/test-utils'
import FinderEmpty from '@/components/Finder/FinderEmpty.vue'

describe('FinderEmpty', () => {
  describe('<template>', () => {
    it('matches previous snapshot', () => {
      const wrapper = shallowMount(FinderEmpty)

      expect(wrapper).toMatchSnapshot()
    })

    it('has correct data-tipe-iu attibute', () => {
      const wrapper = shallowMount(FinderEmpty)

      expect(wrapper.attributes()['data-tipe-ui']).toBe('TipeFinder')
    })
  })
})
