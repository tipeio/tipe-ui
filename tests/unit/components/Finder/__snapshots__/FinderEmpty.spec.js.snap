// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FinderEmpty <template> matches previous snapshot 1`] = `<div data-tipe-ui="TipeFinder">Empty</div>`;
